### DISCUSSION TITLE
Ingestion/headless processing

### DISCUSSION THREAD

**Discussion by joestump** [icereed/paperless-gpt#2] [Ideas]
My Paperless workflow is likely less rigorous than others. I scan directly to it via a WiFi scanner with an ingestion folder and then batch process stuff. I mostly rely on correspondent and document type along with free text search. 

One thing that would be nice is if I could have a similar workflow with paperless-gpt where it's just ingesting documents continuously. Maybe track whether it's been processed with tags? 

Ideally, it'd do something like this:

* Periodically query for unprocessed documents.
* Pull correspondents and document types from the API so they can be fed into the prompt.
* Feed correspondents, document type, tags, and title back to Paperless-ngx. 

There's just no way I'm going to do better than Ollama at this over time. 😅

**Comment by icereed** [comment]
Nice idea. Have you played around with an Ollama model how good it performs with document types, tags and correspondents?
In my case I need to perform some manual corrections when using llama3.1 or similar. Maybe because my documents are German. GPT-4o works quite well without interventions. 

So my question is: in an unattended mode do we need a way to see past modified documents inside the paperless-gpt to review?
